projects/able-player/template/index.html

```diff
@@ -40,7 +40,28 @@
   </head>
 
   <body>
-    <video data-able-player data-skin="2020" preload="auto" playsinline data-youtube-nocookie data-allow-fullscreen data-youtube-id="<%= ancillary.youtube_url %>">
+    <video data-able-player data-skin="2020" preload="auto" playsinline data-youtube-nocookie data-allow-fullscreen
+      data-youtube-id="<%= ancillary.youtube_url %>">
+      <% if (format.captions) { %>
+        <track kind="captions" src="<%= fieldUrl('captions') %>" srclang="en" label="captions" />
+        </track>
+      <% } %>
+      <% if (format.metadata) { %>
+        <track kind="metadata" src="<%= fieldUrl('metadata') %>" srclang="en" label="metadata" />
+        </track>
+      <% } %>
+      <% if (format.chapters) { %>
+        <track kind="chapters" src="<%= fieldUrl('chapters') %>" srclang="en" label="chapters" />
+        </track>
+      <% } %>
+      <% if (format.descriptions) { %>
+        <track kind="descriptions" src="<%= fieldUrl('descriptions') %>" srclang="en" label="descriptions" />
+        </track>
+      <% } %>
+      <% if (format.subtitles) { %>
+        <track kind="subtitles" src="<%= fieldUrl('subtitles') %>" srclang="en" label="subtitles" />
+        </track>
```

Comment:
these track elements are all malformed, they have self closing opening tags and closing tags. examples on mdn and able-player docs both use self-closing track tags, so i think that is probably the way to do it. can remove the closing tags

also apparently "Subtitles" applies to only translations and that is why they require a language attribute. we don't have the ability to set that up in the admin right now (multiple subtitles with different associated languages) so lets skip subtitles for now

Reply:
I removed the redundant closing tags and removed srclang and the subtitles track.